diff --git a/node_modules/electron-windows-msix/static/templates/AppxManifest.xml.in b/node_modules/electron-windows-msix/static/templates/AppxManifest.xml.in
index 202c30c..7b7cded 100644
--- a/node_modules/electron-windows-msix/static/templates/AppxManifest.xml.in
+++ b/node_modules/electron-windows-msix/static/templates/AppxManifest.xml.in
@@ -6,7 +6,7 @@
   xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10"
   xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
   xmlns:desktop2="http://schemas.microsoft.com/appx/manifest/desktop/windows10/2"
-  IgnorableNamespaces="uap uap3 desktop2">
+  IgnorableNamespaces="uap uap3 desktop desktop2">
   <Identity Name="{{IdentityName}}"
           ProcessorArchitecture="{{ProcessorArchitecture}}"
           Version="{{Version}}"
@@ -25,6 +25,8 @@
   <Capabilities>
     <rescap:Capability Name="runFullTrust" />
     <Capability Name="internetClient" />
+    <DeviceCapability Name="webcam" />
+    <DeviceCapability Name="microphone" />
   </Capabilities>
   <Applications>
     <Application Id="App"  Executable="app\{{AppExecutable}}" EntryPoint="Windows.FullTrustApplication">
@@ -35,6 +37,11 @@
           Square150x150Logo="assets\Square150x150Logo.png"
           BackgroundColor="{{PackageBackgroundColor}}">
       </uap:VisualElements>
+      <Extensions>
+        <desktop:Extension Category="windows.startupTask" Executable="app\{{AppExecutable}}" EntryPoint="Windows.FullTrustApplication">
+          <desktop:StartupTask TaskId="AgeofScreenStartup" Enabled="true" DisplayName="{{AppDisplayName}}" />
+        </desktop:Extension>
+      </Extensions>
     </Application>
   </Applications>
 </Package>
